### PR TITLE
Stop indenting lists in trainee dashboard

### PR DESCRIPTION
**Pull request by chrismedrela**
Fixes https://github.com/swcarpentry/amy/issues/855

### CODE REVIEW

**Comment by gvwilson** [comment]
:+1:

File: workshops/static/css/amy.css [hunk 1]
```diff
@@ -200,3 +200,7 @@ table tr label {
     margin-top: 0;
     margin-bottom: 0;
 }
+
+ul.no-left-padding {
```
Comment (pbanaszkiewicz) [inline workshops/static/css/amy.css]: Don't you want to fix all the tables that have `<ul>` elements?

**Comment by pbanaszkiewicz** [comment]
@chrismedrela ping :-)

**Comment by chrismedrela** [comment]
@pbanaszkiewicz pong

**Pull request merged**